src/hooks/__tests__/use-sync-queue.test.ts

```
@@ -0,0 +1,390 @@
+    Object.defineProperty(navigator, "serviceWorker", {
+      value: mockSWContainer,
+      configurable: true,
+      writable: true,
+    });
+  }
+
+  function dispatchSWMessage(data: unknown) {
+    const handlers = swListeners.get("message");
+    if (handlers) {
+      const event = new MessageEvent("message", { data });
+      handlers.forEach((handler) => handler(event));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)